data-serving/data-service/src/controllers/case.ts

```diff
@@ -14,6 +14,9 @@ import { logger } from '../util/logger';
 import stringify from 'csv-stringify/lib/sync';
 import _ from 'lodash';
 
+// TODO: de-jank
+const CaseFieldTextFile = 'https://raw.githubusercontent.com/globaldothealth/list/main/data-serving/scripts/export-data/functions/01-split/fields.txt';
```

Comment:
This can be loaded from the filesystem, right? data service image would need this present

Reply:
Oh yeah